python/tank/template.py

```diff
@@ -562,8 +619,12 @@ def make_template_paths(data, keys, roots):
                             "template should be in the strings section "
                             "instead?" % (template_name, definition))
 
-        root_path = roots[root_name]
-        template_path = TemplatePath(definition, keys, root_path, template_name)
+        root_path = multi_os_roots[root_name][sys.platform]
```

Comment:
Is this safe to assume the dictionary is fully-formed and contains an entry for each platform?  It probably is but just checking!